appveyor.yml

```diff
@@ -13,12 +13,6 @@ platform:
 #   allow_failures:
 #   - julia_version: latest
 
-branches:
-  only:
-    - staging
-    - trying
-    - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags
```

Comment:
keep?

Reply:
If we keep this, that means there will be no CI run on pull requests.

Reply:
I think those are only for branch pushes,not PRs.

Reply:
So according to https://github.com/JuliaLang/Pkg.jl/pull/1401 it looks like it's half-true; we don't get the build of the tip of the PR branch (Which I think is valuable, it helps to debug bad interactions and general flakiness) so perhaps what we should do is to add `/^refs/pull/.*/` or something, so that we get both, but we don't get pushes of branches that don't have a PR open, to keep the number of unnecessary CI runs down?